src/Tizen.Wearable.CircularUI.Forms.Renderer/Shell/ShellRenderer.cs

```diff
@@ -88,56 +92,61 @@ void InitializeNavigationDrawer()
                 return;
             }
 
-            // TODO. Need to improve, Change to high level component, implement without Panel
-            _drawer = new Panel(XForms.NativeParent);
-            _drawer.Show();
-            _drawer.IsOpen = false;
-            _drawer.Direction = PanelDirection.Bottom;
-            _drawer.Toggled += OnDrawerToggled;
-            _drawer.SetScrollable(true);
-            _drawer.SetScrollableArea(1.0);
-            _mainLayout.PackEnd(_drawer);
             _navigationView = new NavigationView(XForms.NativeParent)
             {
                 AlignmentX = -1,
                 AlignmentY = -1,
                 WeightX = 1,
-                WeightY = 1
+                WeightY = 1,
+                BackgroundColor = EColor.Yellow
```

Comment:
Is it a temporary code?, if true please add some comment like `// TODO remove it`

Reply:
Fixed.